_data/types.yaml

```diff
@@ -2,7 +2,7 @@
 
 # team member roles
 
-pi:
+principal-investigator:
```

Comment:
Change your `role` in casey-greene.md to `principal-investigator` because of this change.